test/simplifyExpression/basicsSearch/mixedNumber.test.js

```diff
@@ -0,0 +1,103 @@
+const convertMixedNumberToImproperFraction = require(
+  '../../../lib/simplifyExpression/basicsSearch/convertMixedNumberToImproperFraction');
+const mixedNumber = require('../../../lib/simplifyExpression/basicsSearch/mixedNumber');
+const TestUtil = require('../../TestUtil');
+
+function testIsMixedNumber(input, output) {
+  TestUtil.testBooleanFunction(mixedNumber.isMixedNumber, input, output);
+}
+
+function testIsNegativeMixedNumber(input, output) {
+  TestUtil.testBooleanFunction(mixedNumber.isNegativeMixedNumber, input, output);
+}
+
+function testGetWholeNumber(input, output) {
+  input = TestUtil.flattenAndParse(input);
+  TestUtil.testFunctionOutput(mixedNumber.getWholeNumber, input, output);
+}
+
+function testGetNumerator(input, output) {
+  input = TestUtil.flattenAndParse(input);
+  TestUtil.testFunctionOutput(mixedNumber.getNumerator, input, output);
+}
+
+function testGetDenominator(input, output) {
+  input = TestUtil.flattenAndParse(input);
+  TestUtil.testFunctionOutput(mixedNumber.getDenominator, input, output);
+}
+
+function testConvertMixedNumberToImproperFraction(exprString, outputList, outputStr) {
+  TestUtil.testSubsteps(convertMixedNumberToImproperFraction, exprString, outputList, outputStr);
+}
+
+describe('isMixedNumber', function () {
+  const tests = [
+    ['5(1)/(6)', true],
+    ['19(2)/(3)', true],
+    ['-1(7)/(8)', true],
+    ['4*(1/2)', false],
+    ['(1/2)3', false],
+    ['3*10/15', false],
```

Comment:
So Mathjs fails to parse both of those. I get `SyntaxError: Unexpected part "2" (char 3)`. It's fortunately not a problem for our app because the input always comes in looking like `1(2)/(3)`. Maybe when we switch to Mathparser (some day ... haha) this will work?